morphos/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: morphos_web_1
+      APP_PORT: 8080
+
+  web:
+    image: ghcr.io/danvergara/morphos-server:0.2.0
```

Comment:
We need to add the multi-arch image digest here to make sure all installs and app restarts result in the same image being downloaded.

Since you are using ghcr.io and have a multi-arch build for your image you can find the right digest listed under the tag. Here's the digest for your latest 0.3.0 for example:

<img width="713" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/7c3cfbf8-8d8b-4d5c-803a-c49361c7cd3a">

So for 0.3.0 you would use:
```
ghcr.io/danvergara/morphos-server:0.3.0@sha256:b09a886b6ae1c2c3ab3911d0b3513b1a8c9953e80edeb942d33814700ebadfc7
```

Reply:
I just replaced the old one with the one you suggest.
Thanks!